pkg/metrics/metrics.go

```diff
@@ -7,10 +7,14 @@ import (
 	"github.com/integr8ly/integreatly-operator/pkg/resources"
 	l "github.com/integr8ly/integreatly-operator/pkg/resources/logger"
 	"github.com/integr8ly/integreatly-operator/version"
+	"github.com/prometheus/client_golang/api"
 	prometheusv1 "github.com/prometheus/client_golang/api/prometheus/v1"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/config"
+	"github.com/prometheus/common/model"
```

Comment:
```suggestion
	prometheusApi "github.com/prometheus/client_golang/api"
	prometheusv1 "github.com/prometheus/client_golang/api/prometheus/v1"
	"github.com/prometheus/client_golang/prometheus"
	prometheusConfig "github.com/prometheus/common/config"
	prometheusModel "github.com/prometheus/common/model"
```

nit: maybe use more descriptive imports ? 🤔 

If not, I think we should at least keep imports consistent with `prometheusConfig` as this import name is used in `rhmi_controller.go`